### DISCUSSION TITLE
OpenWRT X64系统内运行失败

### DISCUSSION THREAD

**Discussion by lwq315** [AlistGo/alist#396] [Q&A] [answered]
刚下载到OpenWRT内运行了一下，X64版运行后无输出，也没有监听端口，X32版运行后报./alist-linux-386: line 1: syntax error: unexpected "("

系统：
root@OPENWRT 11:49 alist# uname -a
Linux OPENWRT 5.10.89 #0 SMP Sun Oct 24 09:01:35 2021 x86_64 GNU/Linux

**Comment by xhofe** [comment, marked as answer]
prebuilt使用的都是`glibc-libc`，openwrt使用的是`musl-libc`，所以不兼容，可用使用docker或自行编译。之后可能会增加musl的prebuilt

**Comment by xhofe** [comment]
在v2.0.7中已增加 `musl-libc` 的prebuilt
